lib/OpenQA/Client/Handler.pm

```diff
@@ -39,5 +39,11 @@ sub _build_url {
 
 sub _build_post { $_[0]->client->build_tx(POST => shift()->_build_url(+shift()) => form => +shift()) }
 
+sub is_local {
+    my $self = shift;
+    my $host = $self->_build_url('/')->to_abs->host;
+    return $host eq 'localhost' || $host eq '127.0.0.1';
```

Comment:
Is IPv6 supported? Shouldn't here be '::1' as well?

Reply:
And added.